test/v3/addLiquidityBoosted/addLiquidityBoosted.integration.test.ts

```diff
@@ -172,9 +172,6 @@ describe('Boosted AddLiquidity', () => {
                         addLiquidityBuildCallOutput.minBptOut.amount,
                 ).to.be.true;
             });
```

Comment:
Just to confirm - we're happy native tests are covered enough in addLiquidityPartialBoosted.integration.test.ts?

Reply:
I'd say that from the SDK perspective we should be covered because they reuse the same implementation 👍 
In any case, I'd say we can ship as is and ask SC team to create a test pool with wrapped WETH so we can test on the UI.

Reply:
What do you mean by "wrapped WETH" ? is wwETH a new thing? 

Also friendly reminder we have [pool creation UI](https://pool-creator-git-v3-pool-creation-balancer.vercel.app/?_vercel_share=xpIdz0S1R08JOWZx6BhA38OLthCZyBTb) and [SDK script](https://github.com/balancer/b-sdk/blob/main/examples/createAndInitPool/createAndInitPoolV3.ts) for creating pools

Reply:
ah i see gotta init the buffer within the vault. i wonder if we will have UI for that eventually or if plan is to just create them internally as needed